sphinx/writers/latex.py

```diff
@@ -1252,7 +1252,7 @@ def visit_rubric(self, node):
         if len(node.children) == 1 and node.children[0].astext() in \
            ('Footnotes', _('Footnotes')):
             raise nodes.SkipNode
-        self.body.append('\\paragraph{')
+        self.body.append('\\paragraph*{')
```

Comment:
In this PR, I did not add any migration pass. I suppose nobody want to keep the behavior.